ss_admin_console/main/default/classes/constants/ApexActionType.cls

```diff
@@ -0,0 +1,9 @@
+public with sharing class ApexActionType extends Enumeration {
```

Comment:
Too generic class name. Let's rename it to DataFeederExecutionType.
And then rename static fields to TEST and RUN